backend/baseclass.pm

```diff
@@ -517,8 +517,7 @@ sub format_vtt_timestamp {
     $caption .= " --> ";
     $caption .= sprintf(POSIX::strftime("%T.%%03d\n", gmtime($frametime_ms / 1000)), $frametime_ms % 1000);
     # clock value as caption text
-    $caption .= sprintf(POSIX::strftime("[%FT%T.%%03d]\n", localtime($walltime)), 1000 * ($walltime - int($walltime)));
```

Comment:
This function is for formatting the VTT subtitles. Removing this line will break them. I suppose @StefanBruens had a reason to do the computation this way. The last line is supposed to represent the "wallclock" time (as opposed to the raw frametime).

Reply:
Yeah, this is the wrong line. Have a look at `basetest.pm` @KevinKipkorir